cdap-common/src/main/java/co/cask/cdap/common/twill/AbortOnTimeoutEventHandler.java

```diff
@@ -1,95 +0,0 @@
-/*
- * Copyright © 2014 Cask Data, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-package co.cask.cdap.common.twill;
-
-import com.google.common.collect.ImmutableMap;
-import org.apache.twill.api.EventHandler;
-import org.apache.twill.api.EventHandlerContext;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.util.Map;
-import java.util.concurrent.TimeUnit;
-
-/**
- * A Twill {@link EventHandler} that abort the application if for some runnable it cannot provision container for
- * too long.
- */
-public class AbortOnTimeoutEventHandler extends EventHandler {
```

Comment:
How would the naming work then? It doesn't seem right for `TwillAppLifecycleEventHandler` to extend `AbortOnTimeoutEventHandler` with those names.